src/main/java/javalotto/domain/TotalLottoCount.java

```diff
@@ -0,0 +1,30 @@
+package javalotto.domain;
+
+public class TotalLottoCount {
+
+    private final LottoCount manuallyLottoCount;
+    private final LottoCount automaticallyLottoCount;
+
+    private TotalLottoCount(LottoCount manuallyLottoCount, LottoCount automaticallyLottoCount) {
+        this.manuallyLottoCount = manuallyLottoCount;
+        this.automaticallyLottoCount = automaticallyLottoCount;
+    }
+
+    public static TotalLottoCount of(LottoCount totalLottoCount, LottoCount manuallyLottoCount) {
+        LottoCount automaticallyLottoCount = totalLottoCount.getRemainExceptFor(manuallyLottoCount);
```

Comment:
해당 getRemainExceptFor 메서드의 목적이 궁금합니다!
LottoCount 내에 count getter가 있는 이상 불필요해보여서요~
```suggestion
        LottoCount automaticallyLottoCount = LottoCount.withCount(totalLottoCount.getCount() - manuallyLottoCount.getCount());
```

suggestion을 위해서 불리는 쪽에다가 코멘트를 달았는데요 
LottoCount 입장에서는 getCount을 제공하고 있기도 하고, 각 LottoCount끼리의 연산 로직이 객체의 역할 내의 필수 기능은 아닌것 같아서 제안드려봅니다ㅎㅎ

Reply:
동의합니다!! _**객체에 메시지를 던져서 결과를 얻어온다**_ 라는 문구에 너무 몰입해있어서 값 객체에도 이런 기능을 만들어버렸네요 ㅎㅎ

```java
Integer a = 3;
a.minus(2);
```
이런 메서드가 없듯이, 객체가 값을 던지는 **값 객체**인 이상 저런 기능까진 필요가 없었는데 좀 과했던 것 같습니다! ㅎㅎㅎ